ref/commands/server/server-windowsservice.adoc

```diff
@@ -6,74 +6,88 @@
 //
 // Contributors:
 //     IBM Corporation
+:page-layout: server-command
+:page-type: command
+= server WinService command
 
-= server start command
+Register, unregister, start, and stop the named Open Liberty server as a Windows service.
 
-Starts the named Open Liberty server as a Windows service (Windows only).
+The Windows service name and Windows service display name is `serverName`. After you register, you can find and administer the Open Liberty server as a service by using `regedit` from the Windows Registry program. Entries for the service in the registry are in the following locations:
 
+----
+HKEY_LOCAL_MACHINE->SOFTWARE->Wow6432Node->Apache Software Foundation->Procrun 2.0->serverName
+HKEY_LOCAL_MACHINE->SYSTEM->CurrentControlSet->services->serverName
+----
 
-== Syntax
+== Usage examples
 
-Run the command from `path_to_liberty/bin`.
+Register the `myserver` server as a Windows service:
 
 ----
 server registerWinService serverName
 ----
 
-serverName::
-The name of the server to run the command against. If no server is specified, a server called `defaultServer` is automatically created to run the command against.
-
-The Windows service name and Windows service display name is `serverName`. After you register, you can find and administer the Open Liberty server as a service by using `regedit` from the Windows Registry program. Entries for the service in the registry are in the following locations:
+Unregister the `myserver` server:
 
 ----
-HKEY_LOCAL_MACHINE->SOFTWARE->Wow6432Node->Apache Software Foundation->Procrun 2.0->serverName
-HKEY_LOCAL_MACHINE->SYSTEM->CurrentControlSet->services->serverName
+server unregisterWinService myserver
 ----
 
+Start the `myserver` server as a Windows service:
 
-== Options
+----
+server startWinService myserver
+----
 
-There are no optional parameters for this command.
+Stop the `myserver` server:
 
-== Usage
+----
+server stopWinService myserver
+----
 
-Start `myserver` server in the background:
+Start or stop the `myserver` server by using the server script on Windows systems:
 
 ----
-server start myserver
+server.bat start myserver
```

Comment:
I don't really know why we are using these since they don't relate in any way to the windows service.